cli/spark/aztklib.py

```diff
@@ -7,21 +7,4 @@
 '''
 
 def load_spark_client():
-    secrets_config = config.SecretsConfig()
-    secrets_config.load()
-
-    return aztk.spark.Client(
-        aztk.spark.models.SecretsConfiguration(
-            batch_account_name=secrets_config.batch_account_name,
-            batch_account_key=secrets_config.batch_account_key,
-            batch_service_url=secrets_config.batch_service_url,
-            storage_account_name=secrets_config.storage_account_name,
-            storage_account_key=secrets_config.storage_account_key,
-            storage_account_suffix=secrets_config.storage_account_suffix,
-            docker_endpoint=secrets_config.docker_endpoint,
-            docker_password=secrets_config.docker_password,
-            docker_username=secrets_config.docker_username,
-            ssh_pub_key=secrets_config.ssh_pub_key,
-            ssh_priv_key=secrets_config.ssh_priv_key
-        )
-    )
\ No newline at end of file
+    return aztk.spark.Client(config.load_aztk_screts())
```

Comment:
This file seems less necessary now.

Reply:
yeah

Reply:
I think it could go directly in each of the endpoints. Like here: https://github.com/Azure/aztk/blob/1693f6ef20574afece0ce13f32c0a29697e5bd61/cli/spark/endpoints/cluster_add_user.py#L24

Just replace with `spark_client = aztk.spark.Client(config.load_aztk_screts())`

Reply:
yeah actually could do that, i'll change it